.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,3 +1,9 @@
 #### Checklist
-- [ ] Unit tests
-- [ ] Updated testplan
+- [ ] Functionality meets specifications
+- [ ] Functionality working
```

Comment:
agree with Mark, one checkbox would be enough (or I might be missing the added value of the second ;-) )